src/components/deviceMap/deviceMap.js

```diff
@@ -168,7 +168,7 @@ class DeviceMap extends Component {
   render() {
     const { BingMapKey } = this.props;
     return (
-      <DashboardPanel title={lang.DEVICELOCATION} indicator={this.props.showSpinner} className="map-container">
+      <DashboardPanel title={lang.DEVICELOCATION} initialIndicator={this.props.showInitialSpinner} indicator={this.props.showSpinner} className="map-container">
```

Comment:
Rename `initialIndicator` to `showSpinner` or something. Whether the spinner is being shown at the initial load or afterward isn't really important to the flag.

Reply:
IntialIndicator is shown only on intila load and filter change.